infra/terraform/modules/lambda_mgmt/README.md

```diff
@@ -0,0 +1,60 @@
+## Lambda Management 
+====================
+
+Used to provision lambda functions
+
+Module Input Variables
+----------------------
+
+- `lambda_function_name` - Name for Lambda function
+- `lambda_runtime` - A [valid](http://docs.aws.amazon.com/cli/latest/reference/lambda/create-function.html#options) Lambda runtime environment. Defaults to `go1.x`
+- `zipfile` - Path to zip archive containing Lambda function
+- `handler` - The entrypoint into your Lambda function, in the form of `filename.function_name`. For `Golang` this must match `lambda_function_name`
+- `schedule_expression` - A [valid rate or cron expression](http://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html)
+- `source_code_hash` - The base64 encoded sha256 hash of the archive file - see TF [archive file provider](https://www.terraform.io/docs/providers/archive/d/archive_file.html)
+- `timeout` - (optional) The amount of time your Lambda Function has to run in seconds. Defaults to 3. See [Limits](https://docs.aws.amazon.com/lambda/latest/dg/limits.html)
+- `enabled` - (optional) Boolean expression. If false, the lambda function and the cloudwatch schedule are not set. Defaults to `true`.
+- `tag_key` - (optional) The key of the tag to filter resources on
```

Comment:
I'm not 100% sure about this module (just started to have a look). Is the idea to have this to "provision" different lambda functions? But then why do we have these input variables specific of a certain lambda function (the ELB snapshot one)?

Reply:
Agreed.  Could be more generic.